app/models/userModel.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+        unique: true,
+        allowNull: false
+      },
+
+      name: { type: DataTypes.STRING },
+      surname: { type: DataTypes.STRING, allowNull: false },
```

Comment:
It is better to use `last_name` for this. It is kind of standard for this type of fields